include/ginkgo/core/base/executor.hpp

```diff
@@ -1445,9 +1496,10 @@ class HipExecutor : public detail::ExecutorBase<HipExecutor>,
      * @param master  an executor on the host that is used to invoke the device
      *                kernels
      */
-    static std::shared_ptr<HipExecutor> create(int device_id,
-                                               std::shared_ptr<Executor> master,
-                                               bool device_reset = false);
+    static std::shared_ptr<HipExecutor> create(
```

Comment:
Could you also add the documentation for device_reset and alloc_mode of this function? also Cuda